File: src/main/java/com/seniority/shelter/dataaccess/entites/Place.java
package com.seniority.shelter.dataaccess.entites;


import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
public class Place {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    private String name;
    private String city;
    private String postcode;

    // for JPA only, no use
    public Place() {
    }
}

File: src/main/java/com/seniority/shelter/findPlace/entities/Place.kt
package com.seniority.shelter.findPlace.entities

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table


@Entity(name = "FIND_PLACE")
@Table(name = "PLACE")
data class Place(
    var name: String = "",
    var city: String = "",
    var postcode: String = "",
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    val id: Long = 1,
)
